package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "timezone-mock",
-  "version": "1.2.1",
+  "version": "1.3.4",
```

Comment:
Probably didn't mean to add this file or change this value?  I'll update the version number when it's published to NPM.

Reply:
I stumbled across this change myself and wondered why the version changed.
But in fact it is only the file `package-lock.json` and not `package.json`.
The file `package.json` already contains the current version number `1.3.4`, but the file `package-lock.json` still contains the old version `1.2.1`.
When calling `npm install`, this was adjusted to the correct version, so that now both files contain the same version number again.
So as I understand it, this is not an actual update of the version number and the change should have no effect.
However, I am happy to simply remove the change from the PR.

Reply:
Ah, makes sense, fine to leave it then.  package-lock.json is in a weird place where it doesn't in any way affecting published packages, only local dev, so never gets updated when I bump the version and publish, I guess =).